### PR TITLE
OnApplicationQuit is not reachable

### PR DESCRIPTION
**Pull request by Wenish**
OnApplicationQuit is not reachable after the while loop so i removed it

### CODE REVIEW

File: Assets/ColyseusClient.cs [hunk 1]
```diff
@@ -151,11 +149,4 @@ void OnChangeFallback (PatchObject change)
 //		Debug.Log (change.path);
 //		Debug.Log (change.value);
 	}
-
-	void OnApplicationQuit()
-	{
-		// Make sure client will disconnect from the server
-		room.Leave ();
-		client.Close ();
-	}
```
Comment (endel) [inline Assets/ColyseusClient.cs]: The problem I see here is when you hit the "play" button on the editor, and then stop the game, the connection is not fully closed. The server continues having the client reference there.

**Comment by endel** [comment]
Thanks @Wenish, I've added one comment to your pull-request. I believe the [OnApplicationQuit()](https://docs.unity3d.com/ScriptReference/MonoBehaviour.OnApplicationQuit.html) is still necessary.

**Comment by Wenish** [comment]
@endel oh ur right this is an unity method which gets called automaticly.

so its just the call which is unnecessary

**Comment by endel** [comment]
Thanks a lot @Wenish! 👍 👍 👍

**Pull request merged**